src/Component/RuleCard/RuleCard.tsx

```diff
@@ -94,7 +95,9 @@ export const RuleCard: React.FC<RuleCardProps> = ({
       <Divider type='vertical' />
       <div className='gs-rule-card-content'>
         <h2>{rule.name}</h2>
-        <span><>1:{rule.scaleDenominator?.min || '-'} <MinusOutlined /> 1:{rule.scaleDenominator?.max || '-'}</></span>
+        <span>
+          <>1:{rule.scaleDenominator?.min || '-'} <MinusOutlined /> 1:{rule.scaleDenominator?.max || '-'}</>
```

Comment:
I guess the extra `<>`  can be omitted.

Reply:
Unfortunately not. Removing this makes the linting fail, as a span seems to be expected to only have a single child.